File: 1512-number-of-good-pairs/1512-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        
        total = 0
        for i in range(len(nums)):
            for j in range(len(nums)):
                if nums[i] == nums[j] and i < j:
                    total  = total + 1
        return total

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int total = 0;
        for (int i= 0; i < nums.length;i++){
            for (int j = 0; j < nums.length;j++){
                if (i < j  && nums[i] == nums[j]){
                    total += 1;
                }
            }
        }
        return total;
    }
}